api/agentpb/agent.proto

```diff
@@ -213,6 +216,7 @@ message StartActionRequest {
     PostgreSQLShowCreateTableParams postgresql_show_create_table_params = 6;
     PostgreSQLShowIndexParams postgresql_show_index_params = 7;
     MongoDBExplainParams mongodb_explain_params = 8;
+    PTSummaryParams pt_summary_params = 10;
```

Comment:
9?

Reply:
@BupycHuk When I merged it with latest master 9 was used. google.protobuf.Duration timeout = 9; few lines below.

Reply:
Ok